### DISCUSSION TITLE
File Naming Conventions

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#656] [Q&A]
Hi @shaselton  @shaselton-usds 
For Multiple Plans per File the listed naming standard is YYYY-MM-DD_payer or issuer  name_index.file extenstion. 

Is it acceptable to use a numeric value in the file name when there are multiple plans names for a group health plan? The plan names in the table of contents would show the multiple plan names. 

Examples: 
File Name = 2022-12-01_12345_index.json.gz 

Table of Contents =

    "reporting_entity_name": "XYZ Company",
    "reporting_entity_type": "Third Party Administrator",
    "reporting_structure": [
        {
            "reporting_plans": [
                {
                    "plan_name": "ABC CompanyPPO",
                    "plan_id_type": "EIN",
                    "plan_id": "123456789",
                    "plan_market_type": "group"
                },
                {
                    "plan_name": "DEFCompanyPPO,
                    "plan_id_type": "EIN",
                    "plan_id": "987654321",
                    "plan_market_type": "group"
                },
                           ],
            "in_network_files": [
                {
                    "description": "in-network file",
                    "location": "https://2022-12-01/inNetworkRates/2022-12-01_8afb2c26-aed5-4d1f-ad46-b007210a89f5_XYZ Company.json.gz"
                }
            ],
            "allowed_amount_file": {
                "description": "allowed amount file",
                "location": "https://2022-12-01/allowedAmounts/2022-12-01_8afb2c26-aed5-4d1f-ad46-b007210a89f5_XYZ Company.json.gz"
            }
        },
        {
            "reporting_plans": [
                {
                    "plan_name": "ABC CompanyEPO",
                    "plan_id_type": "EIN",
                    "plan_id": "123456789",
                    "plan_market_type": "group"
                },
                {
                    "plan_name": "DEFCompanyEPO,
                    "plan_id_type": "EIN",
                    "plan_id": "987654321",
                    "plan_market_type": "group"
                }
            ],
            "in_network_files": [
                {
                    "description": "in-network file",
                    "location": "https://2022-12-01/inNetworkRates/2022-12-01_dbc3fabb-2651-41f2-a395-4aa3fe48dee6_XYZ Company.json.gz"
                }
            ],
            "allowed_amount_file": {
                "description": "allowed amount file",
                "location": "https:///2022-12-01/allowedAmounts_dbc3fabb-2651-41f2-a395-4aa3fe48dee6_XYZ Company.json.gz"
            }

       
            }
        }
    ]
}

**Comment by shaselton-usds** [comment]
Hi @Ginger228!

I'm not sure I'm following here, my apologies! Are you asking about the naming convention of the Table of Contents or the naming convention of in-network/allowed-amount files inside the table of contents?

The Table of Contents file name should include the payer/issuer's name and inside you can add as many plans as you want to the `reporting_plans` array.

**Reply by Ginger228** [reply]
> Hi @Ginger228!
> 
> I'm not sure I'm following here, my apologies! Are you asking about the naming convention of the Table of Contents or the naming convention of in-network/allowed-amount files inside the table of contents?
> 
> The Table of Contents file name should include the payer/issuer's name and inside you can add as many plans as you want to the `reporting_plans` array.

@shaselton-usds  apologies I wasn't very clear.   I'm trying to find out if it is acceptable for a group health plan could use their plan # in the file name with the multiple payer names inside the file.   I'm afraid if we only use one name as the group health plan in the the file it will be misleading once a user opens the file and locates subsidiaries of the group health plan. 

Thanks again.
